Tests/XMLCoderTests/SpacePreserveTest.swift

```diff
@@ -51,4 +63,14 @@ final class SpacePreserveTest: XCTestCase {
         ).decode(Item.self, from: nestedXML)
         XCTAssertFalse(item.text?.isEmpty ?? true)
     }
+
+    func testCopyRight() throws {
+        let result = try XMLDecoder().decode(String.self, from: copyrightXML)
+        XCTAssertEqual(result, "Copyright © 2019 Company, Inc.")
+
+        let item = try XMLDecoder().decode(Item.self, from: nestedCopyrightXML)
+        XCTAssertEqual(item, Item(text: "Copyright © 2019 Company, Inc."))
+
+        XCTAssertEqual("Copyright © 2019 Company, Inc.".trimmingCharacters(in: .whitespacesAndNewlines), "Copyright © 2019 Company, Inc.")
```

Comment:
@MaxDesiatov can a repeated string be replaced with a constant?

Reply:
great point, fixed now